tests/Fixtures/Integration/priority/comment_to_phpdoc,phpdoc_to_comment.test

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Integration of fixers: comment_to_phpdoc,phpdoc_to_comment.
+--RULESET--
+{"comment_to_phpdoc": true, "phpdoc_to_comment": true}
```

Comment:
How do you show that order of fixers matters (you put this in priority dir), if there is no input code, means that none of the fixer does any job ?

Reply:
I've added other comment which would change.